content/blog/managing-google-workspace-with-terraform.md

```diff
@@ -0,0 +1,295 @@
+
+## Managing Google Workspace with Terraform
+
+We looked at a couple of open source solutions and decided on using the Terraform [Google Workspace](https://github.com/hashicorp/terraform-provider-googleworkspace) provider. We often use SaaS-specific Terraform providers to provision user accounts for DataDog and other SaaS services, so this was an easy decision.
+
+We heard great things about [GAM](https://github.com/GAM-team/GAM) (an imperative command line tool) from a few colleagues, but we didn't need its full range of capabilities and prefer to use declarative systems so we know the user account and group settings we see in config files are indeed the values in production.
+
+In terms of using the provider, while there are a few Terraform modules out there for managing Google Workspaces, we decided to create our own Terraform module. The motivation to build a new module from scratch stemmed from our desire to:
+
+1. Democratize user onboarding powers so Matt (CEO/CTO) is not a bottleneck.
+2. Provide transparency and history for changes to our team's permissions.
+3. Easily manage SSO configurations allowing engineers to sign into multiple AWS accounts using their Google Identity.
+4. Intuitively organize user and team settings in a clear and straight forward approach.
+
+Here's the GitHub link for the Google Workspace module:  
+[https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation](https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation)
+
+## Getting Started With the Module
+
+To make the module easy to get up and running with your own Google Workspace, we included two practical on-ramps for existing and brand new Google Workspaces:
+
+1. **Import an existing Google Workspace**  
+   We shared the [terraform code](https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation/blob/main/examples/import-existing-org/main.tf) we used to declare existing [users](https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation/blob/main/examples/import-existing-org/users.yaml) and [groups](https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation/blob/main/examples/import-existing-org/groups.yaml) in YAML, and the [import blocks](https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation/blob/main/examples/import-existing-org/imports.tf) to easily import an existing setup. We additionally shared a [Python script](https://github.com/masterpointio/terraform-googleworkspace-users-groups-automation/blob/main/examples/import-existing-org/debugging-script.py) to view the JSON of your existing resources (which you could give to an LLM and tell it to follow the example to create your own `users.yaml` and `groups.yaml` files)..
+
+2. **Create New Resources**  
```

Comment:
I'd make this the same type of capitalization as the point 1 (or change point 1 to have the same as this). 

Does every word in the header get capitalized (as here) or not (as in point 1).